static/App.js

```diff
@@ -78,7 +78,7 @@ function App() {
   }
 
   const Map = () => {
-    const firstResult = (placesData.results && placesData.results[0].geometry.location) || {}
+    const firstResult = (placesData.results && placesData.results.length > 0 && placesData.results[0].geometry.location) || {}
```

Comment:
so many changes

Reply:
These were necessary because the page would error because technically results would be there places.results is true, but in the case where the results were empty this would error.